### PR TITLE
New footer

### PR DESCRIPTION
**Pull request by eballeste**
new footer w/updated sass files

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/517)

### CODE REVIEW

**Comment by jseldess** [comment]
Looks good overall, @eballeste, and I checked all the links. But the columns in the footer seem a little unevenly spaced. Is that intentional? + @jess-edwards 

screen shot 2016-07-27 at 10 17 26 pm

**Comment by jess-edwards** [comment]
@jseldess & @eballeste 
I think this is a trick of the eyes. Because the COMPANY column has the shortest menu items, it appears wider than the others. I've measured, and they're identical (there is the slightest offset on TALK TO US). 
screen shot 2016-07-28 at 8 41 12 am

**Comment by jseldess** [comment]
Ah, thanks for testing that, @jess-edwards. Then all's well! OK if I merge this, or are there still discussions in flight?

File: _includes/footer.html [hunk 1]
```diff
@@ -1,74 +1,90 @@
...
-							<div class="contact_email"><a href="mailto:info@cockroachlabs.com" target="_blank">info@cockroachlabs.com</a></div>
-							-->
-						</div>
-					</div>
-				</div>
+		<div class="row">
+			<div class="company-nav-col col-xs-6 col-sm-4 col-md-15">
+				<ul class="footer-nav">
+					<li class="header">Company</li>
+					<li><a href="http://www.cockroachlabs.com/about/" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-depth-0">About</a></li>
```
Comment (mberhault) [inline _includes/footer.html]: `http://www.cockroachlabs.com` -> `https://www.cockroachlabs.com`
There are a bunch more in this file and others in this PR.

**Comment by jseldess** [comment]
Review status: 0 of 8 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[_includes/footer.html, line 7 \[r3\]](https://reviewable.io:443/reviews/cockroachdb/docs/517#-KNimMx2p5sMu0RMOFY1-r3-7:-KNlzb5VXh08qifHXhue:1283435985) ([raw file](https://github.com/cockroachdb/docs/blob/1d73f01d3c1052ef3a24ea33196aae0c13168649/_includes/footer.html#L7)):*
<details><summary><i>Previously, mberhault (marc) wrote…</i></summary>
> `http://www.cockroachlabs.com` -> `https://www.cockroachlabs.com`
> 
> There are a bunch more in this file and others in this PR.
</details>
Good catch, Mark! Enrique, please change all instances of `http` to `https`. Then we should be good to go.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/517)*

**Comment by eballeste** [comment]
@jseldess Sorry I missed that! Just updated all the links and references to the site including some I did in the css. Should not be an issue anymore.

**Comment by jseldess** [comment]
Great. Thanks, @eballeste. Merging.

**Pull request merged**